src/disk_manager.rs

```diff
@@ -493,16 +463,8 @@ fn safe_to_remove_disk(
     let host_info = h_info.expect("Failed to gather host information");
     debug!("Gathered host info: {:?}", host_info);
 
-    info!("Getting outstanding repair tickets");
-    //let tickets =  in_progress::get_pending_tickets(&db_pool,host_details_mapping.storage_detail_id);
-  //  let region_id = in_progress::get_region_id(&db_pool, &host_info.region)?;
-  //  let storage_id = in_progress::get_storage_id(&db_pool, &host_info.storage_type.to_string())?;
-  //  let storage_detail_id =
-    //    in_progress::get_storage_detail_id(&db_pool, storage_id, region_id, &host_info.hostname)?;
-
     info!("Getting all  outstanding repair tickets");
-   // let tickets = in_progress::get_outstanding_repair_tickets(&db_pool, storage_detail_id)?;
-   let tickets = in_progress::get_AllPendOut_tickets(&db_pool)?;
+    let tickets = in_progress::get_AllPendOut_tickets(&db_pool)?;
```

Comment:
the compiler is going to warn you about this but could you rename this to snake_case?

Reply:
ok let me check

Reply:
Rename using snake_case and tested